src/overlays/actors/ovl_En_Niw_Girl/z_en_niw_girl.c

```diff
@@ -1,4 +1,11 @@
+/*
+ * File: z_en_niw_girl.c
+ * Overlay ovl_En_Niw_Girl
+ * Description: Girl that chases a cuckoo around in Hyrule Market and Kakariko Village
+ */
+
 #include "z_en_niw_girl.h"
+#include "vt.h"
```

Comment:
you have niw girl in this PR. think you forgot to make a new branch

Reply:
I will create a branch EnExRuppy2.  After fixing these I will close this PR and upload the new one.